### DISCUSSION TITLE
Artifacts in HRRR hour 0 data

### DISCUSSION THREAD

**Discussion by stephent** [NOAA-GSL/GSL-Regional-Model-Forum#46] [Q&A] [answered]
Hi all - I've noticed that the hour 0 forecast often has visual artifacts which are not present in subsequent hours. Frequently they appear to be circular or partly circular. Here's an example for the `visibility` layer with the artifacts marked with arrows:

![vis_full_sfc_f000](https://github.com/user-attachments/assets/874eb879-505a-4050-9e20-e7dcf4c3b5db)

Contrast this with hour 01 where no such artifacts are visible (as best I can tell):

![vis_full_sfc_f001](https://github.com/user-attachments/assets/9cee19f1-c7e6-42d4-a4e0-1d62e5ccbe6f)

Is there a known reason for why these appear in hour 00, and are only certain forecast layers affected?

Similar issues are seen in other layers, e.g. low-cloud cover, where the hour 00 data can appear very blocky in contrast to later hours.

**Comment by stephent** [comment]
Hi @EricJames-NOAA - is this the right place for this type of question? Thanks!

**Comment by EricJames-NOAA** [comment, marked as answer]
@stephent yes this is a good place for the question!  Sorry for my late response.  The artifacts you are seeing stem from the use of our non-variational stratiform cloud analysis system used in HRRR.  The system "clears" and "builds" clouds based on available observations of cloud hydrometeors in the 3-D volume, mostly using satellite cloud top height observations and METAR ceilometer observations.  If METAR observations indicate low clouds or fog, cloud hydrometeors are inserted within a 100 km circle of the observation, unless other observations indicate clear conditions in any of that area.  

You can find more details in our paper describing the analysis system:
https://doi.org/10.1175/MWR-D-20-0319.1

**Reply by stephent** [reply]
Perfect - thank you! I have some reading to do now...
